packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -195,6 +197,7 @@ export class GraphQLTransform {
     scope,
     sqlDirectiveDataSourceStrategies,
     synthParameters,
+    deploymentIdentifier,
```

Comment:
as above, should `deploymentIdentifier` be part of `synthParameters` rather than a top-level attribute of `TransformOption`?